src/sdk.ts

```diff
@@ -53,8 +53,10 @@ export type CloudFlavor = {
 /** Saved Connection info for a Stardog instance */
 export type Connection = {
   __typename?: 'Connection';
+  dashboard?: Maybe<Scalars['String']>;
```

Comment:
These were the new fields I talked about, we can expose them in the sdk if you add `dashboard` and `index` to the query:

https://github.com/stardog-union/portal-sdk/blob/main/src/documents/listConnectionsQuery.graphql#L2

Reply:
ah you have to edit these manually, thanks!

Reply:
[`4ab3772` (#3)](https://github.com/stardog-union/portal-sdk/pull/3/commits/4ab377212eaaa92c2a50ea2ef0b2f883506fd118)